Library/Homebrew/os/linux/glibc.rb

```diff
@@ -5,7 +5,7 @@ module Glibc
 
       def system_version
         return @system_version if @system_version
-        version = Utils.popen_read("/usr/bin/ldd", "--version")[/ (\d+\.\d+)/, 1]
+        version = Utils.popen_read("/usr/bin/objdump", "--version")[/ (\d+\.\d+)/, 1]
```

Comment:
You should be able to revert this chunk. Linuxbrew should operate correctly if it's not able to determine the version of `glibc`. It's a bug if not.

Reply:
It crashes on this line. I forget why. Will file.

Reply:
Actually, machomebrew crashes, but linuxbrew seems fine. I don't think it's possible that I was on the wrong fork, but I can't tell if this is an actionable bug:

```
$ git checkout machomebrew/origin
$ brew config
...
OS: Ubuntu 17.10 (artful)
Host glibc: N/A
/usr/bin/gcc: 7.2.0
glibc: N/A
Error: uninitialized constant Formulary::FormulaNamespacea0c71e341d02dc282fdc357aca4c912d::Gcc::GlibcRequirement
Did you mean?  Requirement
               GitRequirement
               MPIRequirement
/home/buck/prefix/brew/Library/Taps/homebrew/homebrew-core/Formula/gcc.rb:51:in `<class:Gcc>'
/home/buck/prefix/brew/Library/Taps/homebrew/homebrew-core/Formula/gcc.rb:1:in `load_formula'
/home/buck/prefix/brew/Library/Homebrew/formulary.rb:27:in `module_eval'
/home/buck/prefix/brew/Library/Homebrew/formulary.rb:27:in `load_formula'
/home/buck/prefix/brew/Library/Homebrew/formulary.rb:47:in `load_formula_from_path'
/home/buck/prefix/brew/Library/Homebrew/formulary.rb:102:in `load_file'
/home/buck/prefix/brew/Library/Homebrew/formulary.rb:93:in `klass'
/home/buck/prefix/brew/Library/Homebrew/formulary.rb:89:in `get_formula'
/home/buck/prefix/brew/Library/Homebrew/formulary.rb:280:in `factory'
/home/buck/prefix/brew/Library/Homebrew/extend/os/linux/system_config.rb:32:in `formula_linked_version'
/home/buck/prefix/brew/Library/Homebrew/extend/os/linux/system_config.rb:44:in `block in dump_verbose_config'
/home/buck/prefix/brew/Library/Homebrew/extend/os/linux/system_config.rb:43:in `each'
/home/buck/prefix/brew/Library/Homebrew/extend/os/linux/system_config.rb:43:in `dump_verbose_config'
/home/buck/prefix/brew/Library/Homebrew/cmd/config.rb:12:in `config'
/home/buck/prefix/brew/Library/Homebrew/brew.rb:100:in `<main>'
```